Makefile

```diff
@@ -21,10 +21,16 @@ USERID  := $(shell id -u)
 GRPID   := $(shell id -g)
 
 .PHONY: bin
-bin:
+bin: windows-bin
```

Comment:
Can we not add this dependency? Some people use `make bin` for local development and are probably not interested in building the binaries for windows. It seems fine to me to keep them as separate targets.

Reply:
Sure. I removed the dependency.